### PR TITLE
Add changes for edge-20.6.1

### PR DESCRIPTION
**Pull request by kleimkuhler**
## edge-20.6.1

This edge release is a release candidate for `stable-2.8`! It introduces several
improvements and fixes for multicluster support.

* CLI
  * Added multicluster daisy chain checks to `linkerd check`
  * Added list of successful gatways in multicluster checks section of `linkerd
    check`
* Controller
  * Renamed multicluster gateway ports to `mc-gateway` and `mc-probe`
  * Fixed Service Profiles routes for `linkerd-prometheus`
* Internal
  * Fixed array handling in the `bin/fmt` script
  * Improved error reporting for scripts in failed CI runs
  * Improved logs and event reporting in CI for all integration test failures
  * Fixed `uname` flags for Darwin in the `bin/lint` script
  * Fixed shellcheck errors in all `bin/` scripts (thanks @joakimr-axis!)
* Helm
  * Added support for `linkerd mc allow`
  * Added ability to disable secret rescources for self-signed certs (thanks
    @cypherfox!)
* Proxy
  * Modified the `linkerd-gateway` component to use the inbound proxy, rather
    than nginx, for gateway; this allows Linkerd to detect loops and propogate
    identity

Signed-off-by: Kevin Leimkuhler <kevin@kleimkuhler.com>

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
 # Changes
 
+## edge-20.6.1
+
+This edge release is a release candidate for `stable-2.8`! It introduces several
+improvements and fixes for multicluster support.
+
+* CLI
+  * Added multicluster daisy chain checks to `linkerd check`
+  * Added list of successful gatways in multicluster checks section of `linkerd
```
Comment (alpeb) [inline CHANGES.md:10]: s/gatways/gateways

**Comment by alpeb** [comment]
Would you add a note under Helm to account for the config map renaming? Something like `Renamed "nginx-configuration" ConfigMap to "linkerd-gateway" (please manually remove the former if upgrading from an earlier multicluster install)`

**Comment by kleimkuhler** [comment]
> Would you add a note under Helm to account for the config map renaming? Something like `Renamed "nginx-configuration" ConfigMap to "linkerd-gateway" (please manually remove the former if upgrading from an earlier multicluster install)`

@alpeb Sure I can add this. I believe this is `linkerd-gateway-config` now. mayankshah1607 made the original change, but that was changed before the last edge.

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+  * Added multicluster daisy chain checks to `linkerd check`
+  * Added list of successful gateways in multicluster checks section of `linkerd
+    check`
+* Controller
+  * Renamed `nginx-configuration` ConfigMap to `linkerd-gateway-config` (please
+    manually remove the former if upgrading from an earlier multicluster
+    install, thanks @mayankshah1607!)
+  * Renamed multicluster gateway ports to `mc-gateway` and `mc-probe`
+  * Fixed Service Profiles routes for `linkerd-prometheus`
+* Internal
```
Comment (adleong) [inline CHANGES.md:18]: I don't have a good rubric for which internal changes should get listed in the release notes but none of these seem significant enough from the user's perspective to include.

Comment (kleimkuhler) [reply CHANGES.md:18]: @adleong I don't usually add the Internal section, but these points all seemed particularly useful to mention for external contributors, especially about CI.

I also wanted to make sure Joakmir had a mention for all the work they have done on the scripts. I don't think it really belongs in another section. How would you feel if I just kept that bullet?

Comment (adleong) [reply CHANGES.md:18]: SGTM

**Pull request merged**